packages/router/src/grpc/view-protocol-server/fmd-parameters.ts

```diff
@@ -7,6 +7,6 @@ export const fMDParameters: Impl['fMDParameters'] = async (_, ctx) => {
   const services = ctx.values.get(servicesCtx);
   const { indexedDb } = await services.getWalletServices();
   const parameters = await indexedDb.getFmdParams();
-  if (!parameters) throw new ConnectError('No FMD parameters', Code.FailedPrecondition);
+  if (!parameters) throw new ConnectError('No FMD parameters', Code.Unavailable);
```

Comment:
Error status changed to the same as rust view service. 
And in general it seems that `unavailable` is better in this case because the client will not know when FMD parameters will be available

Reply:
This code is reserved for this:

```typescript
    /**
     * Unavailable, client should back off and retry.
     */
    Unavailable = 14,
```

Think we should revert to the previous error